scripts/version

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 set -eu
 
-cd "$(dirname "$0")"
+cd "$(dirname "$0")/../"
```

Comment:
Maybe we should use `git rev-parse --show-toplevel`?  Maybe we could use the value of `--srcdir` from `./configure`?

Reply:
Maybe we could, but I don't think we should. To reliably get the location we need something like this after all.
```sh
if ! git -C "$(dirname "$0")/../" rev-parse --show-toplevel >/dev/null 2>&1; then
  exit 1
fi
cd "$(dirname "$0")/../"
```

Reply:
Excuse the braino.